platforms/jvm/plugins-java/src/integTest/groovy/org/gradle/java/compile/daemon/JavaCompilerDaemonReuseIntegrationTest.groovy

```diff
@@ -116,7 +116,8 @@ class JavaCompilerDaemonReuseIntegrationTest extends AbstractCompilerDaemonReuse
     }
 
     def "log messages from a compiler daemon are associated with the task that generates them"() {
-        def buildOperations = new BuildOperationsFixture(executer, temporaryFolder)
+        enableBuildOperationsFixture()
```

Comment:
❌ It seems redundant as calling `enableProblemsApiCheck()` also calls `enableBuildOperationsFixture()`

Reply:
I wanted to keep it redundant, as removing the `enableProblemsApiCheck()` would remove the build fixture. This is more about telling the reader that we _need_ that fixture.